### DISCUSSION TITLE
Simple PHP contact from on Static HTML Website

### DISCUSSION THREAD

**Discussion by procedure1** [vercel/community#4781] [Help] [answered]
### Summary

Hi. Sorry for my cluelessness here. I've used standard webhosting for many years and this is my first time working with a system like Vercel. I don't even know what it is really ha ha. Anyway, I have deployed a very simple HTML website with an HTML/PHP contact form page named 'contact.php'.  I'm used to that working automatically on a standard web host. But on my website now, when a user clicks "Contact" in the navigation, it doesn't open the contact page. It downloads it instead.
I believe I need to install PHP on my project. 
in Settings>Project Settings>Install Command
I entered this text from help docs: 
amazon-linux-extras install php8.2
I have my framework preset set to 'other'.
But I'm getting this error:
'Configuration Settings in the current Production deployment differ from your current Project Settings.'
I also found a help discussion saying to add a json file and a api directory, etc but i had errors with that as well.
 I am mystified by the help docs. They are really written only for people fairly advanced on these subjects.
 Can anyone please give me a clue on the problem?
 Thank you.

### Example

https://patfulmer.com

### Steps to Reproduce

Click on 'contact' in the navigation. It will download the form instead of open the page.

**Comment by amyegan** [comment, marked as answer]
Hi @procedure1! PHP is supported via a [Community runtime](https://vercel.com/docs/functions/serverless-functions/runtimes#community-runtimes). You can [configure it with the functions property](https://vercel.com/docs/functions/configuring-functions/runtime#other-runtimes) in a `vercel.json` file.

More info is available in the PHP Runtime repo: https://github.com/vercel-community/php

**Reply by amyegan** [reply]
Info about configuring projects with vercel.json can be found at https://vercel.com/docs/projects/project-configuration

**Comment by procedure1** [comment]
Thanks very much for your help.
I added the vercel.json:
{
  "functions": {
    "api/*.php": {
      "runtime": "vercel-php@0.6.0"
    }
  }
}
and api/index.php:
<?php
echo "<h2>Hello World</h2>";
phpinfo();
?>
-------
(in the comment above it is not displaying the php code as I've entered it with the php opening tags and the echo command for hello world)
However I'm unclear on how to start php in the system. I tried adding:
amazon-linux-extras install php8.2
to the Build Command and also separately to the Install command in the project settings to no avail. 
The docs say something about using the terminal instead but I have no idea how i would use a terminal for this or why I would need to.
When I go to https://patfulmer.com/api I just get a 404 page.
Thanks you!
